packages/core/src/mpi/normalize-patient.ts

```diff
@@ -0,0 +1,175 @@
+import { Address } from "../domain/medical/address";
+import { Patient, PatientData } from "../domain/medical/patient";
+
+// Define default values for each field
+const defaultValues = {
+  firstName: "john",
+  lastName: "doe",
+  address: [{ addressLine1: "123 main street", city: "anytown", zip: "00000" }],
```

Comment:
[We discussed ](https://github.com/metriport/metriport/pull/1160#discussion_r1437807709)that `123 main street` is a valid address, and also that default values would need to be an array of values for each field, some isolated and some in combination with others (`john` is good, `john doe` is not).

Not to mention that those "deny values" (I suggest we don't call it "default values) can be used on `sandbox`.

Maybe we remove this and just ticket implementing this afterwards, as it seems it needs more analysis?

Reply:
In the PR the handleDefaultValues function call is commented out for that very reason. I left the functions in though. I could delete the functions as well, but I figured to leave them so they could be a starting point for the next implementation, which has a ticket already: https://github.com/metriport/metriport/issues/1404#issue-2066286409

Reply:
Maybe we can create a draft PR for that ticket and have that code there?

I might seem nit-picky, but it's a bad practice to have unused code hanging around. When updating the codebase we might see it and don't know why it's there, and might even update it without checking if it's used.

Reply:
I deleted entirely and added the code to the issue for reference: https://github.com/metriport/metriport/issues/1404